README.md

```diff
@@ -35,6 +35,7 @@ Variables without a default value and not marked optional, *MUST* be defined for
 | GP2GP_AMQP_PASSWORD                  |                           | (Optional) password for the AMQP server
 | GP2GP_AMQP_MAX_REDELIVERIES          | 3                         | The number of times an message will be retried to be delivered to consumer. After exhausting all retires, it will be put on DLQ.<queue_name> dead letter queue
 | GP2GP_MHS_INBOUND_QUEUE              | inbound                   | Name of the queue for MHS inbound
+| GP2GP_MHS_OUTBOUND_URL               |                           | URL of the MHS Outbound Endpoint
```

Comment:
The column default could contain the default value the url has in configuration file.

Reply:
@mattd-kainos should the default be localhost or the mock-mhs-adaptor container url?

Reply:
Might be okay to leave this one due to the length

Reply:
I was thinking about the value from yml: http://localhost:8081/mock-mhs-endpoint